receiver/kafkametricsreceiver/metadata.yaml

```diff
@@ -30,6 +39,13 @@ metrics:
       monotonic: false
       value_type: int
       aggregation_temporality: cumulative
+  kafka.broker.log_retention_hours:
```

Comment:
For all of the metrics being added: The unit shouldn't be included in the metric name. [Source](https://opentelemetry.io/docs/specs/semconv/general/metrics/#units)

Reply:
Thanks for these docs, I have made necessary changes. Please LMK if any of these looks odd.